src/allotropy/parsers/appbio_quantstudio_designandanalysis/appbio_quantstudio_designandanalysis_calculated_documents.py

```diff
@@ -0,0 +1,677 @@
+from collections.abc import Iterator
+from typing import Optional
+
+from allotropy.parsers.appbio_quantstudio.calculated_document import (
+    CalculatedDocument,
+    DataSource,
+)
+from allotropy.parsers.appbio_quantstudio.decorators import cache
+from allotropy.parsers.appbio_quantstudio.views import ViewData
+from allotropy.parsers.appbio_quantstudio_designandanalysis.appbio_quantstudio_designandanalysis_structure import (
+    WellItem,
+)
+from allotropy.parsers.utils.uuids import random_uuid_str
+
+
+@cache
+def build_quantity(well_item: WellItem) -> Optional[CalculatedDocument]:
+    if (quantity := well_item.result.quantity) is None:
+        return None
+
+    return CalculatedDocument(
+        uuid=random_uuid_str(),
+        name="quantity",
+        value=quantity,
+        data_sources=[
+            DataSource(feature="cycle threshold result", reference=well_item),
+            DataSource(feature="Y-intercept", reference=well_item),
+            DataSource(feature="Slope", reference=well_item),
+        ],
+    )
+
+
+def build_quantity_mean(
+    view_data: ViewData[WellItem], sample: str, target: str
+) -> Optional[CalculatedDocument]:
+    well_items = view_data.get_leaf_item(sample, target)
+    if (quantity_mean := well_items[0].result.quantity_mean) is None:
+        return None
+
+    data_sources = []
+    for well_item in well_items:
+        quantity_ref = build_quantity(well_item)
+        if quantity_ref is None:
+            return None
+
+        data_sources.append(
+            DataSource(
+                feature="quantity",
+                reference=quantity_ref,
+            )
+        )
+
+    return CalculatedDocument(
+        uuid=random_uuid_str(),
+        name="quantity mean",
+        value=quantity_mean,
+        data_sources=data_sources,
+    )
+
+
+def build_quantity_sd(
```

Comment:
Is this exactly the same as the above function, just with "quantity sd" as the name and referencing `well_items[0].result.quantity_sd` instead of `well_items[0].result.quantity_mean`? Doesn't need to be this PR, but I think it'd benefit from a refactor to a shared `build_quantity_calculated_document(view_data, sample, target, name, get_calculation: Callable[[WellItem], float])` (or `calculation_property: str` but that gives up some type safety)